### ISSUE TITLE
ssl verify failed on Windows

### ISSUE THREAD

**Issue by tomkoenecke** [dlesbre/bibtex-autocomplete#4]
My previous issue was resolved with the following commits and version 1.1.1. Thank you for that.

However, there is more. Now that the .bib file is opened properly, SSL Certificate problems arise.
The log looks like this:
```
==== Reading files =============================================================
Reading file 1 / 1 from 'References.bib'
Read 62 entries from 1 file

==== Completing entries ========================================================
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[unpaywall] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[crossref] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[unpaywall] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[crossref] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[unpaywall] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[researchr] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)
[crossref] WARNING: connection error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)

```

Again, this happens only on Windows (Python 3.9).

Modifying the `HTTPSConnection` in `https.py` to
`connection = HTTPSConnection(domain, timeout=self.connection_timeout, context = ssl._create_unverified_context())` 
fixes this, as described in [this](https://stackoverflow.com/questions/48908044/how-to-disable-ssl-verification-for-http-client-httpsconnection-class-in-python) stackexchange post.

**Comment by dlesbre** [comment]
I'm not a big fan of bypassing SSL verification, especially given those websites should all have valid certificates.

From what I've found online the `certifi` package provides Mozilla's root certificates, can you try running `pip install --upgrade certifi` and then running the script again ?

If that fixes the issue I'l make a prettier error message mentionning this solution and add it as a recommended dependency to the README. Otherwise I can add SSL bypass when on Windows

**Comment by dlesbre** [comment]
I pushed a fix in version 1.1.2.

In the end I added a new CLI flag `-S` or `--ignore-ssl` that uses `ssl._create_unverified_context()`, but the default behavior is still to use a verified context. I also edited the error message to mention you can either use this flag or update your certificates with `pip install --update certifi`.
